opentelemetry-proto/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## vNext
 
 - Bump MSRV to 1.70 [1864](https://github.com/open-telemetry/opentelemetry-rust/pull/1874)
+- **BREAKING** Group log and Span batch by their resource and instrumentation scope before exporting [#1873](https://github.com/open-telemetry/opentelemetry-rust/pull/1873).
```

Comment:
not sure if qualifies as breaking change. We previously emitted valid OTLP. We still do valid OTLP.

Reply:
Agree, updated.

Reply:
The "breaking" warning is still there. Missed a commit push?

Reply:
I thought it was to remove the breaking change from the otel-otlp changelog. otel-proto has the breaking change - as any library directly using it would need to adjust its code).

Reply:
I am not sure if there is any breaking change here.. 
I can see there is a new method for grouping logs by resource/scope, but nothing existing is removed, so existing consumers would be unaffected. They may *optionally* use the new method, if they prefer to have more efficiency.

Reply:
yes true. Removed the breaking tag.